src/odemis/driver/xt_client.py

```diff
@@ -2106,10 +2109,10 @@ def _updateSettings(self):
         super(MultiBeamScanner, self)._updateSettings()
         # Polling XTtoolkit settings
         try:
-            pitch = self.parent.get_pitch() * 1e-6
-            if pitch != self.pitch.value:
-                self.pitch._value = pitch
-                self.pitch.notify(pitch)
+            delta_pitch = self.parent.get_delta_pitch()
+            if delta_pitch != self.deltaPitch.value:
+                self.deltaPitch._value = delta_pitch * 1e-6
```

Comment:
The `* 1e-6` should be on the first line, as it used to be. Otherwise the comparison is going to be always detecting a change.

Reply:
This actually passed in the test_case because we used 0 as a test value. I will also update that value.